tests/integration/test_logcollector/test_command_monitoring/test_command_execution_dbg.py

```diff
@@ -59,99 +59,61 @@
-     'info': 'does not end'},
-    {'log_format': 'full_command', 'command': 'ls -R /tmp', 'alias': '', 'info': 'long_output'},
-    {'log_format': 'full_command', 'command': 'cat doesntexists.txt', 'alias': '', 'info': 'that_fails'},
-    {'log_format': 'full_command', 'command': 'cat "ñ", "テスト", "ИСПЫТАНИЕ", "测试", "اختبار".txt', 'alias': '',
-     'info': 'special_chars_filename'},
-    {'log_format': 'full_command', 'command': 'テ``ñスト, ИСПЫТА´НИЕ",\'测`试", "اختبا', 'alias': '',
-     'info': 'special_chars_command'},
-    {'log_format': 'full_command', 'command': 'echo ***', 'alias': '', 'info': 'special_chars_echo'},
-]
-
-if sys.platform == 'linux':
-    parameters.append({'LOG_FORMAT': 'command', 'COMMAND': 'timeout 2 tail -f /dev/random', 'ALIAS': 'killed_by_test'})
-    parameters.append({'LOG_FORMAT': 'full_command', 'COMMAND': 'timeout 2 tail -f /dev/random', 'ALIAS': ''})
-    parameters.append({'LOG_FORMAT': 'command', 'COMMAND': 'ss -l -p -u -t -4 -6 -n', 'ALIAS': ''})
-    parameters.append({'LOG_FORMAT': 'full_command', 'COMMAND': 'ss -l -p -u -t -4 -6 -n', 'ALIAS': ''})
-    metadata.append({'log_format': 'command', 'command': 'timeout 2 tail -f /dev/random', 'alias': '',
-                     'info': 'killed_by_test'})
-    metadata.append({'log_format': 'full_command', 'command': 'timeout 2 tail -f /dev/random', 'alias': '',
-                     'info': 'killed_by_test'})
-    metadata.append({'log_format': 'command', 'command': 'ss -l -p -u -t -4 -6 -n', 'alias': '',
-                     'info': 'many_arguments'})
-    metadata.append({'log_format': 'full_command', 'command': 'ss -l -p -u -t -4 -6 -n', 'alias': '',
-                     'info': 'many_arguments'})
-
-configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
-configuration_ids = [f"{x['log_format']}_{x['info']}" for x in metadata]
-
-
-# fixtures
-@pytest.fixture(scope="module", params=configurations, ids=configuration_ids)
-def get_configuration(request):
-    """Get configurations from the module."""
-    return request.param
+def remove_item_from_list(item):
+    """Remove item from configuration, metadata and case_ids list.
+
+    Returns:
+        Index of the next element to be checked.
+    """
+    del t1_configuration_metadata[item]
+    del t1_configuration_parameters[item]
+    del t1_case_ids[item]
+    return item - 1
+
+
+# Accepted values test configurations (t1)
+t1_configuration_parameters, t1_configuration_metadata, t1_case_ids = get_test_cases_data(t1_cases_path)
+
+if sys.platform != 'linux':
+    index = 0
+    for _ in range(len(t1_configuration_metadata)):
+        if t1_configuration_metadata[index]['os'] == 'linux' and index > 0:
+            index = remove_item_from_list(index)
+        index += 1
+
+t1_configurations = load_configuration_template(t1_configurations_path, t1_configuration_parameters,
+                                                t1_configuration_metadata)
```

Comment:
According to this logic block:
- Do the non-Linux OS launch all cases?
- Maybe is better to use a test case file for each OS (according to the test differences)

Reply:
Done in [here](https://github.com/wazuh/wazuh-qa/pull/3486/commits/fa745773ca16d8f5751f9a9f5d0d3c4c3ccc40a1)